h/schemas/forms/admin/group.py

```diff
@@ -24,56 +22,65 @@
 VALID_GROUP_TYPES = (("restricted", _("Restricted")), ("open", _("Open")))
 
 
-@colander.deferred
-def group_creator_validator(node, kw):
```

Comment:
I think `colander.deferred` was only being used here to access `kw`, but the values here are available as `form.bindings` which is also much more informative. This matches the other parts where we don't use `deferred` to access these variables.